CElegans/pythonScripts/c302/runAndPlot.py

```diff
@@ -22,24 +23,47 @@ def run_c302(config,
              plot_ca=True,
              plot_connectivity=False,
              param_overrides={},
-             config_package=""):
-    
+             config_param_overrides={},
```

Comment:
@lungd Which PR is adding config_param_overrides to c302_Full.py etc?

Reply:
@pgleeson https://github.com/openworm/CElegansNeuroML/pull/94/files#diff-52e22c8c1456975b5cb29048a7814ad0R12.